cli/src/lib/deploy.ts

```diff
@@ -76,7 +78,12 @@ export const DEPLOY_FLAGS = {
   }),
 };
 
-type DealState = "notCreated" | "notMatched" | "matched" | "ended";
+type DealState =
```

Comment:
Maybe `enum` would be more preferable here?

Reply:
enum and union in this situation function exactly the same way from type-safety stand point except with enum you will have to add `DealState` prefix and import each time you want to check this state. I mostly prefer unions for this reason. Enums in my opinion are only appropriate if you have to put some weird constant behind the name (e.g. like we have with `DealStatus`)